src/invidious/yt_backend/connection_pool.cr

```diff
@@ -45,24 +42,22 @@ struct YoutubeConnectionPool
 
   private def build_pool
     DB::Pool(HTTP::Client).new(initial_pool_size: 0, max_pool_size: capacity, max_idle_pool_size: capacity, checkout_timeout: timeout) do
-      conn = HTTP::Client.new(url)
-      conn.family = CONFIG.force_resolve
+      conn = make_client(url, force_resolve: true)
       conn.family = Socket::Family::INET if conn.family == Socket::Family::UNSPEC
```

Comment:
I think this was here to prevent `CONFIG.force_resolve` to pass `Socket::Family::UNSPEC` and hence should be moved to `make_client()` too!